StoryMap/StoryMap/Screens/StoryDetail/ViewModel/StoryDetailViewModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  StoryDetailViewModel.swift
+//  StoryMap
+//
+//  Created by Dory on 02/11/2021.
+//
+
+import Foundation
+
+protocol StoryDetailViewModelType: AnyObject {
+    var story: Story { get }
```

Comment:
I would keep the story object private and expose only the properties that the view controller really needs, such as title and image data.

Reply:
done